pkgs/applications/science/biology/sumatools/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchurl, gcc, zlib }:
+
+let
+  nativeBuildInputs = [ gcc ];
+  meta = with stdenv.lib; {
+    description = "Fast and exact comparison and clustering of sequences";
+    homepage = http://metabarcoding.org/sumatra;
+    maintainers = [ maintainers.bzizou ];
+    platforms = platforms.unix;
+  };
+
+in rec {
+
+  # Suma library
+  sumalibs = stdenv.mkDerivation rec {
+      version = "1.0.34";
+      name = "sumalibs";
+      src = fetchurl {
+        url = "https://git.metabarcoding.org/obitools/${name}/repository/sumalib_v${version}/archive.tar.gz";
+        sha256 = "07xk5s0ii1nf97ihryzk22012ichd3w16ccv9c5jpar0g92rn0kx";
+      };
```

Comment:
You should use `fetchFromGitLab` with a custom `domain` (see https://github.com/NixOS/nixpkgs/blob/bad5d145c28db9005d2fa8056ff41a146c0c6bf1/pkgs/applications/backup/deja-dup/default.nix#L12-L18 for example) instead. The straightforward way to download an archive from gitlab instances has proved problematic in the past (see https://github.com/NixOS/nixpkgs/issues/39200)